src/actions/images.js

```diff
@@ -53,6 +53,9 @@ export const createImage = (...args) => {
           type: types.CREATE_IMAGE_SUCCESS,
           payload: response.data,
         });
+        api.myImages().then(json => {
```

Comment:
Might be worth moving this to it's own method if you're using it in multiple places but passing dispatch around is kind of a pain.